Makefile

```diff
@@ -1,5 +1,18 @@
+# Some important environment variables
 EMULATOR ?= simh
 
+# Sometimes you _really_ need to use a different `touch` or `rm`.
+TOUCH ?= touch
+MKDIR ?= mkdir
+EXPECT ?= expect
+MKDIR ?= mkdir
+RM ?= rm
+LN ?= ln
+SED ?= sed
+TEST ?= test
+GIT ?= git
+CAT ?= cat
+
 include conf/network
 
 # The directores listed in SRC, DOC, and BIN are put on the sources tape.
```

Comment:
Use `$()` rather than `${}` in the definitions of KLH10, etc.?

Reply:
I wrote ${PWD} a long time ago since I thought that would use the environment variable PWD rather than a makefile variable.  @ams alluded to such a difference.  Maybe this is outside the scope of this pull request.

Reply:
Probably; it was changed to `$(PWD)` in the rp0.dsk rule, though. In GNU make they mean the same thing - you could also use `$(CURDIR)` instead of the environment variable.

Reply:
@aap also wanted $(CURDIR) for some reason, so I changed to that througout.